### ISSUE TITLE
Nothing is focused when a layer is removed using the layer control

### ISSUE THREAD

**Issue by Anshpreet8** [Maps4HTML/Web-Map-Custom-Element#589]
Tab to the remove layer button on the layer control and press enter.

Expected: the next element should be focused
Actual: Nothing is focused

**Comment by Malvoz** [comment]
>Expected: the next element should be focused

I guess the next element refers to the next layer item?

Another consideration is when all layers have been removed, what should be focused then? The map perhaps?

**Comment by Anshpreet8** [comment]
> Another consideration is when all layers have been removed, what should be focused then? The map perhaps?

Yes, the map makes sense. Or potentially the links at the bottom right as the tab order is the layer control then links?

**Comment by Malvoz** [comment]
I think the attribution links are inherently a bit confusing because there's no context to what they're for:

https://github.com/Maps4HTML/Web-Map-Custom-Element/issues/247#attribution-links:

> - [ ] The container element for the attribution should provide contextual information, e.g. `role="group" aria-label="Map data attribution"` (or a control to toggle attribution should be labelled, per #216).

But even so, I agree that's the logical thing to do. 👍🏼
